src/ui/app/server.js

```diff
@@ -0,0 +1,31 @@
+require('dotenv').config()
```

Comment:
Any changes here that needed to happen to properly handle cookies? Or was I just hitting issues with env variables not being properly set?

Reply:
I think it was the env variables issue. Once I fixed this, everything worked as expected for me.